Example/Auth/Tests/FIRPhoneAuthProviderTests.m

```diff
@@ -314,7 +314,9 @@ - (void)testVerifyPhoneNumberUIDelegate {
       NSMutableString *fakeRedirectURLString =
           [NSMutableString stringWithString:kFakeRedirectURLStringWithoutReCAPTCHAToken];
       [fakeRedirectURLString appendString:kFakeReCAPTCHAToken];
-      [[_mockAuth authURLPresenter] canHandleURL:[NSURL URLWithString:fakeRedirectURLString]];
+
+      OCMExpect([_mockAuth canHandleURL:OCMOCK_ANY]).andReturn([[_mockAuth authURLPresenter]
```

Comment:
The `canHandleURL:` seems to get called too early.

Reply:
Done.